cmd/kola/options.go

```diff
@@ -64,7 +65,7 @@ func init() {
 	ss("debug-systemd-unit", []string{}, "full-unit-name.service to enable SYSTEMD_LOG_LEVEL=debug on. Specify multiple times for multiple units.")
 	sv(&kola.UpdatePayloadFile, "update-payload", "", "Path to an update payload that should be made available to tests")
 	sv(&kola.Options.IgnitionVersion, "ignition-version", "", "Ignition version override: v2, v3")
-
+	ssv(&kola.BlacklistedTests, "blacklist-test", []string{}, "Test to blacklist. Use multiple times to blacklist multiple tests")
```

Comment:
For the description let's keep it simple with `List of tests to blacklist`.

`cobra` also allows comma-seperated values (https://godoc.org/github.com/spf13/pflag#StringSliceVar) so unless you feel it's necessary to spell out one possible way of using it we might as well keep it open.